csrank/objectranking/object_ranker.py

```diff
@@ -141,11 +142,10 @@ def __call__(self, X, *args, **kwargs):
         """
         return self.predict(X, **kwargs)
 
-    def set_init_lr_callback(self, callbacks):
+    def set_initial_lr_for_scheduler(self, callbacks):
         for c in callbacks:
             if isinstance(c, LRScheduler):
                 c.initial_lr = K.get_value(self.optimizer.lr)
-                self.logger.info("Setting lr {} for {}".format(c.initial_lr, c.__name__))
         return callbacks
```

Comment:
But what if the user is not using this callback at all?

Reply:
Then there is no problem. The learning rate of the optimizer can also be tuned for the rankers separately.

Reply:
Then we will have two separate Lr values if we are using two lr as tunable and also even if the user is not providing the optimizer lr as tunable for the the optimizer it will still set the learning rate initial value to the optimizer learning rate

Reply:
https://github.com/keras-team/keras/blob/ef13db05731bfd53fa0a877637c99c1734be933b/keras/callbacks.py#L626
We can simply remove initial_lr as the new keras API is able to handle it by itself for setting the lr for the callback same as that for the optimizer